docs/openstack/backend_services_deployment.md

```diff
@@ -120,7 +126,7 @@ podified OpenStack control plane services.
     name: openstack
   spec:
     secret: osp-secret
-    storageClass: local-storage
+    storageClass: $STORAGE_CLASS_NAME
```

Comment:
you need to update the glance template on line 159 to use this too

 glance:
    template:
      storageClass: $STORAGE_CLASS_NAME